### PR TITLE
feat: added axiosGet,updateuser,button files

### PR DESCRIPTION
**Pull request by SrujanaGajagouni**
Screenshot 2024-07-24 130727
### On clicking the button it displays updated user
Screenshot 2024-07-24 130740
### Displaying fields using axios get
Screenshot 2024-07-24 130758
### Displaying buttons using map
Screenshot 2024-07-24 130812

### CODE REVIEW

File: Evaluation/AxiosTask.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      } catch (err) {
+        console.log('Failed to fetch posts');
+      }
+    };
+
+    fetchPosts();
+  }, []);
+
+  return (
+    <div>
```
Comment (Jinankhan) [inline Evaluation/AxiosTask.tsx:21]: use mui box and list component here

File: Evaluation/ButtonTask.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    { label: "Save", count: 2 }
+  ];
+
+  return (
+    <>
+        {types.map((type, index) => (
+            Array.from({ length: type.count }).map((_, i) => (
+            <Button 
+                variant='contained' 
+                key={`${index}-${i}`}
```
Comment (Jinankhan) [inline Evaluation/ButtonTask.tsx:15]: use id's instead of indexes for keys

File: Evaluation/constants.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+export interface Posts {
+  id: number;
+  title: string;
+  body: string; 
+}
+
+
+
+export const getPosts = () => axios.get<Posts[]>(API_URL);
```
Comment (Jinankhan) [inline Evaluation/constants.tsx:14]: add the getPosts in a separate service folder.

**Review by Jinankhan — CHANGES_REQUESTED** [review]

File: Evaluation/constants.tsx (hunk 3 above)
Comment (SrujanaGajagouni) [reply Evaluation/constants.tsx:14]: resolved all the comments

File: ReactReview/service/axiosInstance.tsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const axiosInstance = axios.create({
+  baseURL: 'https://jsonplaceholder.typicode.com',
```
Comment (Jinankhan) [inline ReactReview/service/axiosInstance.tsx:4]: put the url in the constants file

File: ReactReview/service/axiosInstance.tsx [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const axiosInstance = axios.create({
```
Comment (Jinankhan) [inline ReactReview/service/axiosInstance.tsx:3]: rename the file to API.ts , use ts for the files that don't return any jsx...

File: ReactReview/service/postService.tsx [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import axiosInstance from './axiosInstance';
+
+export interface Post {
```
Comment (Jinankhan) [inline ReactReview/service/postService.tsx:3]: interface name should be prefixed with I

**Review by Jinankhan — APPROVED** [review]

**Pull request merged**